compiler/test/compilable/stdcheaders.c

```diff
@@ -30,7 +30,11 @@ float x = NAN;
 #include <setjmp.h>
 #endif
 
+#ifndef __aarch64__
+#ifndef __linux__ // /usr/include/linux/types.h(12): Error: __int128 not supported
```

Comment:
I understand the `__aarch64__` one, but isn't `#ifndef __linux__` disabling it for all off linux?

Reply:
That's right, I was making a *or* not and *and* from the conditions. There was a line below in which the check was carried out properly, I don't know why I ignored it.